docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -4225,6 +4225,60 @@ expect(foo.something?).to be_truthy
 * https://rspec.rubystyle.guide/#predicate-matchers
 * https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/PredicateMatcher
 
+== RSpec/ProperlyItSpecify
+
+|===
+| Enabled by default | Safe | Supports autocorrection | Version Added | Version Changed
+
+| Pending
+| Yes
+| Yes
+| <<next>>
+| -
+|===
+
+Check for `specify` and `it` are used properly.
+
+Use specify if the example doesn't have a description,
+use it for examples with descriptions.
+
+=== Examples
+
+[source,ruby]
+----
+# bad
+it do
```

Comment:
We already cover this in `ExampleWithoutDescription` (it even handle one-line  examples and empty string as description).

What do you think of doing just a SpecifyWithDescription cop ?

Reply:
I had missed this. But the default behavior is different from the RSpec Style Guide.
Is it more like `ExampleWithoutDescription` is a cop that wants to offend against an empty string being passed?

Reply:
`it` was divided into the following:
- https://github.com/rubocop/rubocop-rspec/pull/1805